x/masterchef/module_simulation.go

```diff
@@ -23,7 +23,11 @@ var (
 )
 
 const (
-// this line is used by starport scaffolding # simapp/module/const
+	opWeightMsgTogglePoolEdenRewards = "op_weight_msg_toggle_pool_eden_rewards"
+	// TODO: Determine the simulation weight value
+	defaultWeightMsgTogglePoolEdenRewards int = 100
```

Comment:
anyone can help figure this value before merging please?

Reply:
this is fine, it is autogenerated and is same for other modules too

Reply:
Yes I sent different unit tests to the new gov proposal. However, I believe that the migrations and simulation shouldn't be part of the coverage.